packages/bindings/binding.gyp

```diff
@@ -4,9 +4,9 @@
     'sources': [
       'src/serialport.cpp'
     ],
-    'include_dirs': [
-      '<!(node -e "require(\'nan\')")'
-    ],
+    'include_dirs': ["<!(node -p \"require('node-addon-api').include_dir\")"],
+    'dependencies': ["<!(node -p \"require('node-addon-api').gyp\")"],
+    "defines": ["NAPI_DISABLE_CPP_EXCEPTIONS"],
```

Comment:
1. Why not use C++ exceptions?
2. From `node-addon-api` [docs](https://github.com/nodejs/node-addon-api/blob/main/doc/setup.md):
   > If you decide to use `node-addon-api` without C++ exceptions enabled, please consider enabling node-addon-api safe API type guards to ensure the proper exception handling pattern:
   > ```gyp
   > 'defines': [ 'NODE_ADDON_API_ENABLE_MAYBE' ],
   > ```

Reply:
Thanks @cinderblock, I've enabled the CPP exceptions in the latest set of changes